samples/XCT.Sample/Pages/Extensions/ExtensionsGalleryPage.xaml

```diff
@@ -1,9 +1,15 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
-<pages:BasePage xmlns="http://xamarin.com/schemas/2014/forms"
+﻿<?xml version="1.0" encoding="UTF-8" ?>
+<pages:BasePage x:Class="Xamarin.CommunityToolkit.Sample.Pages.Extensions.ExtensionsGalleryPage"
+                xmlns="http://xamarin.com/schemas/2014/forms"
                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                 xmlns:xct="http://xamarin.com/schemas/2020/toolkit"
-                xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages"
-                x:Class="Xamarin.CommunityToolkit.Sample.Pages.Extensions.ExtensionsGalleryPage">
-    <pages:BasePage.Content>
-    </pages:BasePage.Content>
-</pages:BasePage>
+                xmlns:pages="clr-namespace:Xamarin.CommunityToolkit.Sample.Pages">
+
+    <StackLayout Margin="30"
+                 Spacing="30">
+        <Label Text="Showing an embeded image by it resource Id using ImageResource extension"/>
+        <Image WidthRequest="150"
+               HeightRequest="150"
+               Source="{xct:ImageResource Id=Xamarin.CommunityToolkit.Sample.Images.logo.png}"/>
+    </StackLayout>
+</pages:BasePage>
```

Comment:
The gallery page should contain the list of items pointing to extension samples (similar to Converters gallery or behaviors one etc.)

Reply:
Oh my bad good catch, I didn't even noticed that thanks :)